pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -9812,12 +9824,13 @@ final: prev:
 
   rest-nvim = buildNeovimPlugin {
```

Comment:
`rest-nvim` build fails probably because it uses `luaPackages.rest-nvim`, which is not yet updated.

Reply:
If it is not really possible to update `luaPackages.rest-nvim`, maybe drop related to the plugin changes manually?